README.md

```diff
@@ -50,6 +54,29 @@ sls deploy
 ```
 5. 👀 Head to the CloudWatch section of the AWS Console to check out your new dashboards 📊 and alarms ⏰ !
 
+
+## Getting Started - CloudFormation Macro
+
+1. Deploying to your account (via the console)
+Go to the [AWS Application Repository](https://serverlessrepo.aws.amazon.com/applications) and click the Deploy button.
+
+2. To add the Macro to a SAM template,  add it in the **Transform** section :
+```
+Transform: [ "CfMacroSlicWatch"]
+```
+3. 🪛 _Optionally_, add some configuration for the plugin to the `Metadata -> slicWatch` section of `template.yml`.
+Here, you can specify a reference to the SNS topic for alarms. This is optional, but it's usually something you want
+so you can receive alarm notifications via email, Slack, etc.
+
+```
+Metadata:
+  slicWatch:
```

Comment:
We will need to include an example of customising a `Function` with `Metadata` too. We have examples of this for the serverless plugin, so we might need to adapt the configuration section to illustrate both cases.

Reply:
updated documentation.